src/Microsoft.ML.TensorFlow/doc.xml

```diff
@@ -7,27 +7,29 @@
         Extracts hidden layers' values from a pre-trained Tensorflow model.
       </summary>
       <remarks>
-        The TensorflowTransform extracts the specified output from the operation computed on the graph (given the input(s)) using a pre-trained <a href="https://www.tensorflow.org">Tensorflow</a> model.
-        The transform takes as input the Tensorflow model together with the names of the inputs to the model and name of the operation for which output values will be extracted from the model.
+        The TensorflowTransform extracts the specified outputs from the operations computed on the graph (given the input(s)) using a pre-trained <a href="https://www.tensorflow.org">Tensorflow</a> model.
+        The transform takes as input the Tensorflow model together with the names of the inputs to the model and names of the operations for which output values will be extracted from the model.
 
         This transform requires the <a href="https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.ML.TensorFlow/0.5.0-preview-26830-5">Microsoft.ML.TensorFlow</a> nuget to be installed.
-		
+
         The TensorflowTransform has following assumptions regarding the input, output and processing of data.
         <list type="number">
           <item>
```

Comment:
><item> [](start = 10, length = 6)

not part of your PR, but i have had trouble with list items not displaying if not inside <description>